src/utils.py

```diff
@@ -33,6 +34,16 @@ def ensure_chrome_profile():
         logger.debug(f"Created Chrome profile directory: {chromeProfilePath}")
     return chromeProfilePath
 
+def ensure_firefox_profile():
```

Comment:
can you move both methods from utils.

line 26 and line 37

Reply:
Yeah, the profiles are now belonging to different browser class.